INSTALL.md

```diff
@@ -2,54 +2,18 @@
 
 ### Requirements:
 - PyTorch 1.7
```

Comment:
@LoreggiaGiacomo  it would seem you removed requirements from this list.
Don't do that. Add the versions but do not remove requirements from this list

Reply:
I didn't remove anything from the list: I've added the versions to the modules that are present in the list (except for cocoapi, which is not even in requirements.txt so I don't know its version), plus I added a reference to the requirements.txt file. What is removed is the section 1, as we agreed.

Reply:
Ok, good, github interface confused me. I think COCO API are pycocotools

Reply:
It seems pycocotools is contained inside COCO API and that COCO API contains also other things.
https://github.com/cocodataset/cocoapi/tree/master/PythonAPI/pycocotools
From the repository it seems that the other scripts contained there are not for python and that pycocotools is the only part of COCO API to be used for python. Hence, since we are using just python in this repository, I assume I can substitute "cocoapi" in the requirements list with "pycocotols"?

Reply:
yes

Reply:
Done